ci-operator/step-registry/hypershift/mce/agent/create/hostedcluster/hypershift-mce-agent-create-hostedcluster-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: hypershift-mce-agent-create-hostedcluster
+  cli: latest
+  env:
+  - name: HYPERSHIFT_HC_RELEASE_IMAGE
+    default: ""
+    documentation: "Release image used for the HostedCluster. Empty by default it will use release:latest"
```

Comment:
@csrwng Are we updating the Hypershift `latest` tag with the real latest from main?

Reply:
Sorry I don’t use it here and will remove it. So it will use the default payload of hypershift create chain